app/assets/stylesheets/_screen.scss

```diff
@@ -177,6 +177,14 @@ div.story textarea {
   resize: none;
 }
 
+.description {
+  background: $lightgrey-12;
+  border: 0.5px solid lightgray;
```

Comment:
Normally in this cases, you can put just like this `.5px`

Reply:
Thanks for the tip 😄